packages/pds-ember/app/styles/pds/utils/_a11y.scss

```diff
@@ -13,6 +13,7 @@ $_module: "PDS.Utils.A11y";
   // content from the a11y tree
   width: 1px !important;
   height: 1px !important;
+  clip: rect(0, 0, 0, 0) !important;
```

Comment:
I think you might want to add the clip-path and white-space rules as described here: https://medium.com/web-dev-survey-from-kyoto/the-visually-hidden-technique-303f8e2bd409 (note there are some rules you can clean up and remove too).

Reply:
@junyper Thanks. I've updated the PR. 

I'm skipping the IE6/7 fallback for `clip`  because we (thankfully) don't support these browsers.